pkg/autodiscovery/listeners/common.go

```diff
@@ -31,6 +31,13 @@ const (
 	tagKeyService = "service"
 )
 
+// containerFilters holds container filters for AD listeners
+type containerFilters struct {
```

Comment:
I think you can implement the method `containerFilters.IsExcluded(...)` that wraps the different call to the  `global`, `metrics` or `logs` `containers.Filter` something like

```golang
type (cf *containerFilters) IsExcluded(type containers.FilterType, cInspect.Name, image, "", string) bool {

}
```

Reply:
yes, done ✅